packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/scale/scale.routing.js

```diff
@@ -0,0 +1,31 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'pci.projects.project.kubernetes.details.nodepools.scale',
+    {
+      url: '/scale?nodePoolId',
+      views: {
+        modal: {
+          component: 'pciProjectKubernetesNodePoolsScaleComponent',
+        },
+      },
+      layout: 'modal',
+      translations: {
+        value: ['.'],
+        format: 'json',
+      },
+      params: {
+        nodePoolId: null,
+        nodePoolName: null,
+      },
+      resolve: {
+        goBack: /* @ngInject */ (goToNodePools) => goToNodePools,
+        nodePool: /* @ngInject */ (Kubernetes, kubeId, nodePoolId, projectId) =>
+          Kubernetes.getNodePool(projectId, kubeId, nodePoolId),
+        nodePoolId: /* @ngInject */ ($stateParams) => $stateParams.nodePoolId,
```

Comment:
`$stateParams` is obsolete, use `$transition$.params()` instead please

Reply:
Copy past from another module.
Thanks for your attention.

Fixed.